features/steps/project/issues/issues.rb

```diff
@@ -219,7 +219,7 @@ class Spinach::Features::ProjectIssues < Spinach::FeatureSteps
   end
 
   step 'The code block should be unchanged' do
-    expect(page).to have_content("```\nCommand [1]: /usr/local/bin/git , see [text](doc/text)\n```")
+    expect(page).to have_content("Command [1]: /usr/local/bin/git , see [text](doc/text)")
```

Comment:
```
@javascript
Scenario: Blocks inside comments should not build relative links
  Given I visit issue page "Release 0.4"
  And I leave a comment with code block
  Then The code block should be unchanged
```

```
page.within(".js-main-target-form") do
  fill_in "note[note]", with: "```\nCommand [1]: /usr/local/bin/git , see [text](doc/text)\n```"
  click_button "Add Comment"
  sleep 0.05
end
```

This action can not fail, the fill content is correct.

And that test case title is "Blocks inside comments should not build relative links", seem like means "not change links", not code block `\`\`\``